dnf-behave-tests/features/steps/file.py

```diff
@@ -66,6 +74,19 @@ def file_contains(context, filepath):
     return
 
 
+@behave.step('file "{filepath}" does not contain lines')
+def file_contains(context, filepath):
+    regexp_lines = context.text.split('\n')
+    full_path = os.path.join(context.dnf.installroot, filepath.lstrip("/"))
+    ensure_directory_exists(os.path.dirname(full_path))
+    read_str = read_file_contents(full_path)
+    for line in regexp_lines:
+        if re.search(line, read_str):
+            print("line: " + line + " found")
+            raise AssertionError("File %s contains: \n%s" % (filepath, read_str))
```

Comment:
It looks like that the function does opposite to file_contains. It raises if text was found. Is it correct or did I missed something?

Reply:
The name doesn't matter for these behave.step functions, only the text in the decorator does. In many cases, the function name is just "step_impl". But I will rename it, so that it's less confusing.